frontend/src/components/pages/volunteer/posting/VolunteerPostingDetails.tsx

```diff
@@ -59,11 +61,10 @@ const VolunteerPostingDetails = (): React.ReactElement => {
     history.push(route);
   };
 
-  return !loading && !postingDetails ? (
+  return (!loading && !postingDetails) || error ? (
```

Comment:
```suggestion
  return (!loading && !postingDetails) || postingDetailError ? (
```
maybe make this more explicit?